QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeContainerTote.java

```diff
@@ -24,30 +23,24 @@ public AutonomousModeContainerTote() {
     @Override
     protected void runAutonomous() throws InterruptedException,
             AutonomousModeOverException {
-        // Loading
+        // Pickup container.
         pickupContainer(nudge.get());
         setClampHeight(1.0f);
```

Comment:
Don't hard-code the 1.0f. This should be tunable, and should probably be less than full height because the forklift is slow.